packages/examples/package.json

```diff
@@ -34,6 +31,6 @@
     "typescript": "^3.2.4"
   },
   "dependencies": {
-    "@scalecube/scalecube-microservice": "^0.1.3-alpha.4"
+    "@scalecube/scalecube-microservice": "snapshot"
```

Comment:
I think we also have to use the same version of the package, that is specified in local package.json of @scalecube/scalecube-microservice